src/modules/ProductDetailsPage/ProductNotFoundPage/ProductNotFoundPage.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "@/style.scss";
+
+.not_found_product {
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+    background-color: $original_hover_and_bg;
+    background-image: url('/public/img/product-not-found.png');

```

Comment:
please replace `/public/img/product-not-found.png` to `/img/product-not-found.png`